bin/ghe-restore

```diff
@@ -285,6 +285,16 @@ fi
 # Get GHES release version in major.minor format
 RELEASE_VERSION=$(ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config --get core.package-version' | cut -d '.' -f 1,2)
 
+# If the backup being restored is from an appliance with Actions disabled, restoring it onto an appliance with Actions enabled will cause
+# mismatches in the secrets needed for Actions which ultimately results in Actions not working properly. Note: xargs is to remove whitespace
+ACTIONS_ENABLED_IN_BACKUP=$(git config -f $GHE_RESTORE_SNAPSHOT_PATH/settings.json app.actions.enabled | xargs)
```

Comment:
Do you still need `xargs`?

Reply:
I see you mentioned this above, so we can discuss it there and you can ignore this comment!

Reply:
I'm curious about this, if you have a chance could you run without xargs but with the `-x` flag set? 
You can just do `bash -x ghe-restore` or something... you should be able to see what exactly this is set to. 

Alternatively, you could just run this command in your terminal `git config -f $GHE_RESTORE_SNAPSHOT_PATH/settings.json some.missing.key | wc -c` 

Basically, I wouldn't expect a missing value to give you anything but an empty string and I'm wondering if this is a sign of some other bug!

Reply:
(I'm happy to try this out myself but I don't actually know where `$GHE_RESTORE_SNAPSHOT_PATH` points to so I can't reproduce) 😄

Reply:
`wc -c` returns 7 spaces and a `0`.  `bash -x ghe-restore` instead returns this instead (including only the last part before my `exit 1` that I put there for testing.

```
++ git config -f /Users/lucavallin/Projects/backup-utils-tainted/data/20220801T095510/settings.json --bool app.actions.enabled
+ ACTIONS_ENABLED_IN_BACKUP=
+ cleanup failed
+ '[' -n failed ']'
+ update_restore_status failed
+ false
+ echo failed
+ ghe-ssh lucavallin-0c579e4255abff420.ghe-test.ninja:122 -- 'sudo sponge '\''/data/user/common/ghe-restore-status'\'' >/dev/null'
+ false
+ true
+ ghe-ssh --clean
```

Reply:
Very interesting! I think defending against whitespace is probably appropriate in this case, then 👍 . 
For my own curiosity, would you mind sending me a copy of your settings.json file? 😄

Reply:
Of course, will send it to you via Slack!